### DISCUSSION TITLE
app.yml missing in cypress 11

### DISCUSSION THREAD

**Discussion by tironci** [sorry-cypress/sorry-cypress#687] [Q&A] [answered]
Hi,

I upgraded to Cypress 11 and seems like /Cypress/resources/app/packages/server/config/app.yml is missing and the CLI one-liner to change cypress configuration for all installed versions of cypress is failing. Anyone has seen this issue and know how to fix it?

Thank you

**Comment by agoldis** [comment, marked as answer]
https://currents.dev/readme/guides/cypress-11

**Reply by stokrattt** [reply]
Hi, 
and if I don't use @currents/cli (or cy2 and @currents/nx), I use only sorry-cypress (dashboard, director and api) then I have other options to run sorry cypress for 11 version?
Thanks

**Reply by agoldis** [reply]
I am afraid not, unless you want / can edit a binary file - they have changed the package and removed the config file altogether. It is now 'embedded' into the binary file which is being loaded as-is into memory.

If you can think of a good way to modify the config, please share with me!
